scripts/auto-rebase/rebase.py

```diff
@@ -92,32 +110,40 @@ def make_sure_rebase_script_created_new_commits_or_exit(git_repo, base_branch):
 
 
 def rebase_script_made_changes_considered_functional(git_repo, base_branch):
+    """
+    Returns True if the changes made by the 'rebase.sh' script are
+    considered functional, False otherwise.
+    """
     logging.info(f"Deciding if PR should be created by diffing against {base_branch} branch")
     diffs = git_repo.active_branch.commit.diff(base_branch)
     logging.info(f"Following files changed: {[ d.a_path for d in diffs ]}")
 
-    for d in diffs:
-        if 'scripts/auto-rebase/' in d.a_path:
-            logging.info(f" - {d.a_path} - ignoring")
+    for d_diff in diffs:
```

Comment:
What rule led to this change?

Reply:
`C0103: Variable name "d" doesn't conform to snake_case naming style (invalid-name)` 

`diff_itr` or `d_diff` were two substitutes that came to my mind, and chose to use `d_diff`.

Reply:
Reverted the changes and added `--variable-naming-style=any` to mute snake_case style.